knack/experimental.py

```diff
@@ -50,8 +51,7 @@ def __init__(self, cli_ctx, object_type='', target=None, tag_func=None, message_
         """
 
         def _default_get_message(self):
-            return "This {} is experimental and not covered by customer support. " \
```

Comment:
I removed "not covered by customer support", as for Knack there is no customer support at all!